include/turtle/matcher.hpp

```diff
@@ -91,9 +93,9 @@ namespace mock
         explicit matcher( const Functor& f )
             : c_( f )
         {}
-        bool operator()( BOOST_RV_REF(Actual) actual )
+        bool operator()( typename detail::ref_arg< Actual >::type actual )
         {
-            return c_( boost::forward< Actual >( actual ) );
+            return c_( ::mock::detail::move_if_not_lvalue_reference< typename detail::ref_arg< Actual >::type >( actual ) );
```

Comment:
I know this might not be very clever but at the moment the project uses `mock::…` i.e. without prefixing with `::`.